providers/azure/aci.go

```diff
@@ -1298,8 +1298,26 @@ func getProbe(probe *v1.Probe) (*aci.ContainerProbe, error) {
 
 	var httpGET *aci.ContainerHTTPGetProbe
 	if probe.Handler.HTTPGet != nil {
+		var portValue int
+		port := probe.Handler.HTTPGet.Port
+		switch port.Type {
+		case intstr.Int:
+			portValue = port.IntValue()
+		case intstr.String:
+			portValue = -1
```

Comment:
Is 0 a valid port value? Can we use the 0 value if not?

Reply:
👍